rasa/core/policies/unexpected_intent_policy.py

```diff
@@ -135,148 +146,154 @@ class UnexpecTEDIntentPolicy(TEDPolicy):
+            # Fraction of trainable weights in internal layers.
+            CONNECTION_DENSITY: 0.2,
+            # If 'True' apply dropout to sparse input tensors
+            SPARSE_INPUT_DROPOUT: True,
+            # If 'True' apply dropout to dense input tensors
+            DENSE_INPUT_DROPOUT: True,
+            # If 'True' random tokens of the input message will be masked. Since there is no
+            # related loss term used inside TED, the masking effectively becomes just input
+            # dropout applied to the text of user utterances.
+            MASKED_LM: False,
+            # ## Evaluation parameters
+            # How often calculate validation accuracy.
+            # Small values may hurt performance, e.g. model accuracy.
+            EVAL_NUM_EPOCHS: 20,
+            # How many examples to use for hold out validation set
+            # Large values may hurt performance, e.g. model accuracy.
+            EVAL_NUM_EXAMPLES: 0,
+            # If you want to use tensorboard to visualize training and validation metrics,
+            # set this option to a valid output directory.
+            TENSORBOARD_LOG_DIR: None,
+            # Define when training metrics for tensorboard should be logged.
+            # Either after every epoch or for every training step.
+            # Valid values: 'epoch' and 'batch'
+            TENSORBOARD_LOG_LEVEL: "epoch",
+            # Perform model checkpointing
+            CHECKPOINT_MODEL: False,
+            # Specify what features to use as sequence and sentence features.
+            # By default all features in the pipeline are used.
+            FEATURIZERS: [],
+            # List of intents to ignore for `action_unlikely_intent` prediction.
+            IGNORE_INTENTS_LIST: [],
+            # Tolerance for prediction of `action_unlikely_intent`.
+            # For each intent, the tolerance is the percentage of
+            # negative training instances (trackers for which
+            # the corresponding intent is not the correct label) that
+            # would be ignored by `UnexpecTEDIntentPolicy`. This is converted
+            # into a similarity threshold by identifying the similarity
+            # score for the (1 - tolerance) percentile of negative
+            # examples. Any tracker with a similarity score below this
+            # threshold will trigger an `action_unlikely_intent`.
+            # Higher values of `tolerance` means the policy is more
+            # "tolerant" to surprising paths in conversations and
+            # hence will result in lesser number of `action_unlikely_intent`
+            # triggers. Acceptable values are between 0.0 and 1.0 (inclusive).
+            TOLERANCE: 0.0,
+            # Split entities by comma, this makes sense e.g. for a list of
+            # ingredients in a recipe, but it doesn't make sense for the parts of
+            # an address
+            SPLIT_ENTITIES_BY_COMMA: SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE,
```

Comment:
can please also add `POLICY_PRIORITY: UNLIKELY_INTENT_POLICY_PRIORITY` and `POLICY_MAX_HISTORY: None` to the defaults? (I forgot that for `TEDPolicy`)